jobs/release.Jenkinsfile

```diff
@@ -140,12 +140,17 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
 
             if ((basearch in ['aarch64', 'x86_64']) && params.AWS_REPLICATION) {
                 // Replicate AMI to other regions.
-                stage("Replicate ${basearch} AWS AMI") {
-                    withCredentials([file(variable: 'AWS_CONFIG_FILE',
-                                          credentialsId: 'aws-build-upload-config')]) {
-                        shwrap("""
-                        cosa aws-replicate --build=${params.VERSION} --arch=${basearch} --log-level=INFO
-                        """)
+                stage("${basearch} AWS AMI Replication") {
+                    def credentials = [file(variable: 'AWS_CONFIG_FILE',
+                                            credentialsId: 'aws-build-upload-config'),
+                                       file(variable: "AWS_GOVCLOUD_IMAGE_UPLOAD_CONFIG",
+                                            credentialsId: "aws-govcloud-image-upload-config")]
+                    for (cred in credentials) {
+                        tryWithCredentials(cred) {
```

Comment:
There's a subtle semantic change here. Before, `AWS_REPLICATION` being checked meant that we required AWS creds to be available. Now, that's no longer the case. I think that's fine, but it's worth calling out (and ideally splitting into a separate preliminary commit).

Reply:
I'll rework it a bit.